### PR TITLE
[WIP] feat: Add Tooltip component

### PR DESCRIPTION
**Pull request by Crash--**
Added `<Tooltip>` component based on `Popover` component.

I'm using it this way in Drive : 
```jsx 
export const Avatar = props => {
  return (
    <Tooltip title={props.textId}>
      <CozyUIAvatar {...props} className={styles['recipient-avatar']} />
    </Tooltip>
  )
}
```

### CODE REVIEW

File: react/Tooltip/Tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    dataTip,
+    dataFor
+  })
+  return (
+    <React.Fragment>
+      {newChild}
+      <ReactTooltip
+        place="bottom"
+        effect="solid"
+        className={styles['shared-tooltip']}
```
Comment (Crash--) [inline react/Tooltip/Tooltip.jsx]: I know I've to open the API to place / effect etc.

File: react/Tooltip/tooltip.styl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.shared-tooltip
```
Comment (ptbrowne) [inline react/Tooltip/tooltip.styl]: I think "shared" can be removed

File: react/Tooltip/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import Tooltip from './Tooltip'
+export default Tooltip
```
Comment (ptbrowne) [inline react/Tooltip/index.js]: Nit: `export default from './Tooltip'` ?

File: react/Tooltip/Tooltip.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import ReactTooltip from 'react-tooltip'
+import styles from './tooltip.styl'
+//Only needed for Preact?
```
Comment (ptbrowne) [inline react/Tooltip/Tooltip.jsx]: If it is sure, let's remove the question mark :)

**Comment by ptbrowne** [comment]
Some small remarks, overall LGTM ! Thanks !

**Comment by ptbrowne** [comment]
It would be good to have an example in the styleguide.

There is the styleguide.config.js to change and a Readme.md file to add.

File: react/Tooltip/Tooltip.jsx (hunk 1 above)
Comment (ptbrowne) [reply react/Tooltip/Tooltip.jsx]: I did not understand ^

File: react/Tooltip/tooltip.styl [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    padding     .5rem
```
Comment (GoOz) [inline react/Tooltip/tooltip.styl]:
```suggestion
    padding     rem(8)
```

Comment (ptbrowne) [reply react/Tooltip/tooltip.styl]: Why ?

Comment (GoOz) [reply react/Tooltip/tooltip.styl]: Consistency, there's `rem()` everywhere else

**Comment by Crash--** [comment]
I just pushed a new implementation of `Tooltip` based on our new `Popover` component.  No more external dependencies and no more Fragment hack. 

Added a render props to Popover to know if the component has pop bellow or not. 

@GoOz If you want to style it, feel free to do it ;)

**Comment by GoOz** [comment]
What is it supposed to look like? Can we see an example in styleguidist?

**Comment by Crash--** [comment]
Will reopen one day

**Pull request closed without merging**